backend/tests/docker/test_executor_home_acceptance_scripts.py

```
@@ -353,6 +356,65 @@ def test_remote_docker_acceptance_fails_explicitly_without_docker():
+            """
+        ),
+        encoding="utf-8",
+    )
+    path.chmod(0o755)
+
+
+def test_rpc_probe_keeps_stdin_open_until_async_response(tmp_path: Path):
+    executor = tmp_path / "delayed-executor.py"
+    _write_delayed_fake_executor(executor)
+
+    result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)